### PR TITLE
Frontend tweaks

### PR DESCRIPTION
**Pull request by jayres**
There are some admin changes from the front end but there is also one major change to the backend with regard to adding a field in the DB for the admin action label

### CODE REVIEW

File: frontend/app/App.js [hunk 1]
```diff
@@ -41,7 +41,7 @@ class Container extends Component {
     return (
       <div>
         <header className="main-header">
-          <Link to="/">CallParty</Link>
+          <Link to="/"><img src="http://callparty.org/assets/images/callparty.png" alt="Call Party Logo" /></Link>
```
Comment (mhfowler) [inline frontend/app/App.js]: nice

**Comment by mhfowler** [comment]
guessing you'll be adding the admin label stuff to the backend as a separate PR?

**Comment by jayres** [comment]
Dammit - I guess I forgot to add those files to this PR. I will send those in and then we can re-review.

File: frontend/css/main.sass [hunk 2]
```diff
@@ -167,3 +170,6 @@ textarea, input[type=text], input[type=password]
     margin-bottom: 20px;
     a 
       margin-left: 10px;
+  img
```
Comment (oshoham) [inline frontend/css/main.sass]: could you make a css class for the header image instead of applying the styles to all img tags?

**Comment by jayres** [comment]
Out of date! Will add a new PR with better code.

**Pull request closed without merging**